tests/flows/validator_churn.go

```diff
@@ -14,10 +14,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-const (
-	nodesPerSubnet = 5
-	newNodeCount   = 5
-)
+const newNodeCount = 2
```

Comment:
Just noticing this now, but `newNodeCount` is a bit of a misnomer. It represents the number of nodes added to the validator set, not the number of validators after the operation.

Reply:
ah I can see that interpretation now that you mention it. I had interpreted it as "count of new nodes", not "new count of nodes". definitely ambiguous!